src/components/Place/PlaceCard/PlaceCard.tsx

```diff
@@ -62,6 +66,18 @@ export default React.memo(function PlaceCard(props: PlaceCardProps) {
     }
   }, [place])
 
+  const handleClickCard = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault()
+    if (props.search) {
+      track(SegmentPlace.PlaceCardClick, {
+        place: place?.id,
```

Comment:
I think we use place for the position in the site, like `world_list`, that would be `place_id` and maybe the place should came from props ¿?